eva-accession-clustering-automation/clustering_automation/create_clustering_progress_table.py

```diff
@@ -138,12 +138,17 @@ def fill_num_rs_id_for_taxonomy_and_assembly(mongo_source, private_config_xml_fi
 
 
 def get_assembly_list_for_taxonomy(private_config_xml_file, taxonomy):
-    assembly_list = []
+    assembly_list = set()
     with get_metadata_connection_handle("development", private_config_xml_file) as pg_conn:
         query = f'SELECT assembly_accession from evapro.assembly where taxonomy_id = {taxonomy}'
```

Comment:
question for my better understanding. 
I used to think **evapro.assembly** is like a master reference table which contains all the assemblies, but that does not seem true now, if there are assemblies present in **eva_progress_tracker.clustering_release_tracker** and not in **evapro.assembly**

Reply:
The metadata database capture all assemblies that go through our submission system. So until we started remapping that was true. 
Now that we are remapping there are assemblies that never received submissions that will have submitted and clustered variants.

Reply:
Thanks for the clarification